Client/Application/AppDelegate.swift

```diff
@@ -62,28 +62,37 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     private func setUserAgent() {
         let webView = UIWebView()
-        let userAgent = webView.stringByEvaluatingJavaScriptFromString("navigator.userAgent")!
-
-        // Extract the WebKit version and use it as the Safari version.
-        let webKitVersionRegex = NSRegularExpression(pattern: "AppleWebKit/([^ ]+) ", options: nil, error: nil)!
-        let match = webKitVersionRegex.firstMatchInString(userAgent, options: nil, range: NSRange(location: 0, length: count(userAgent)))
-        if match == nil {
-            println("Error: Unable to determine WebKit version")
-            return
-        }
-        let webKitVersion = (userAgent as NSString).substringWithRange(match!.rangeAtIndex(1))
+        let currentiOSVersion = floor(NSFoundationVersionNumber)
```

Comment:
I don't think `NSFoundationVersionNumber` is the right thing to check. I'm looking at the headers and the value is the same for `NSFoundationVersionNumber_iOS_6_0` and `NSFoundationVersionNumber_iOS_6_1`. That is very suspicious because I'm pretty sure that WebKit got an upgrade between the two.

I think  `UIDevice.systemVersion` is probably better. Assuming it contains a version number with 3 components (like 6.3.1).